packages/gatsby/src/utils/adapter/__tests__/__snapshots__/manager.ts.snap

```diff
@@ -146,7 +146,7 @@ Array [
   },
   Object {
     "functionId": "static-index-js",
-    "path": "/api/static/",
+    "path": "/api/static",
```

Comment:
Seems like those snapshot changes are incorrect ( https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/91766/workflows/85a3583b-de8f-486a-a473-70b21d1cc4ad/jobs/1111387/tests )

Reply:
The snapshots were passing locally at my EOD, so I was going to take a closer look today at the cause here.